constants/__init__.py

```diff
@@ -27,13 +29,20 @@
 SUBNET_UID = 9
 # The root directory of this project.
 ROOT_DIR = Path(__file__).parent.parent
-# The maximum bytes for the hugging face repo (5 Gigabyte).
-MAX_HUGGING_FACE_BYTES = 5 * 1024 * 1024 * 1024
+# Block at which 7b models, 2048 sequence lengths, bfloat16, and flash attention are used.
+BLOCK_7B = 9_999_999
```

Comment:
Will confirm a specific block prior to merging.

(Also note some of the other constants in this file for this commit get refactored in other commits. Apologies for any lack of clarity).